doc/tutorials/ios-app.md

```diff
@@ -102,45 +102,52 @@ This creates and opens the empty `WORKSPACE` file.
 To build applications for Apple devices, Bazel needs to pull the latest
 [Apple build rules](https://github.com/bazelbuild/rules_apple)
 from its GitHub repository. To enable this, add the following
-[`git_repository`](https://bazel.build/reference/be/workspace#git_repository)
+[`http_archive`](https://bazel.build/rules/lib/repo/http#http_archive)
 rules to your `WORKSPACE` file:
```

Comment:
```suggestion
statements to your `WORKSPACE` file:
```

There is only one `http_archive` rule, then there is a lot of loads. Do we need to link to `http_archive`?